### PR TITLE
Make verbose default for c builds, add quiet option (#1623)

### PR DESCRIPTION
**Pull request by islandblue**
Plus some make-up on do.py. Should still be easy enough to distinguish which changes are related to the issue.

### CODE REVIEW

File: do.py [hunk 1]
```diff
@@ -274,8 +295,10 @@ def install_postprocessing(package_install_option = []):
     expand_file(join('code-postprocessing', 'setup.py.in'),
                 join('code-postprocessing', 'setup.py'),
                 {'COCO_VERSION': git_version(pep440=True)})
-    # copy_tree('code-postprocessing/latex-templates', 'code-postprocessing/cocopp/latex-templates')
-    python('code-postprocessing', ['setup.py', 'install'] + package_install_option, verbose=_verbosity,
+    # copy_tree('code-postprocessing/latex-templates',
+    #           'code-postprocessing/cocopp/latex-templates')
```
Comment (nikohansen) [inline do.py]: One of the cases where obeying PEP 8 makes the code less readable, I would say.

File: do.py [hunk 2]
```diff
@@ -831,6 +855,11 @@ def verbose(args):
     main(args)
     _verbosity = False
 
+def quiet(args):
```
Comment (nikohansen) [inline do.py]: As a general rule, any function should have a docstring. Here, as single line would suffice, but it should be there.

**Comment by nikohansen** [comment]
I don't understand why the appveyor test is labeled as failed? Otherwise the pull can be merged.

**Comment by islandblue** [comment]
Thank you @nikohansen, I added the docstrings and tried to exercise common sense to choose which lines to unwrap and which to keep.

File: do.py [hunk 3]
```diff
@@ -175,14 +170,10 @@ def build_c_integration_tests():
...
                 {'COCO_VERSION': git_version(pep440=True)})
-    copy_file('code-experiments/src/bbob2009_testcases.txt',
-              'code-experiments/test/integration-test/bbob2009_testcases.txt')
-    copy_file('code-experiments/src/bbob2009_testcases2.txt',
-              'code-experiments/test/integration-test/bbob2009_testcases2.txt')
-    make("code-experiments/test/integration-test", "clean",
-         verbose=_build_verbosity)
-    make("code-experiments/test/integration-test", "all",
-         verbose=_build_verbosity)
+    copy_file('code-experiments/src/bbob2009_testcases.txt', 'code-experiments/test/integration-test/bbob2009_testcases.txt')
```
Comment (nikohansen) [inline do.py]: sorry, my comment was regarding PEP 8 and _incommented_ code, otherwise line breaks are OK

**Pull request merged**